lighthouse-core/report/html/report-styles.css

```diff
@@ -59,14 +60,14 @@
   --color-white: #FFFFFF;
 
   /* Context-specific colors */
-  --color-average-secondary: var(--color-orange-700);
   --color-average: var(--color-orange);
```

Comment:
we're following slightly different rules for alphabetical sort, you can just copy/paste this:

```
--color-average-secondary: var(--color-orange-700);
--color-average: var(--color-orange);
--color-fail-secondary: var(--color-red-700);
--color-fail: var(--color-red);
--color-hover: var(--color-gray-50);
--color-informative: var(--color-blue-900);
--color-pass-secondary: var(--color-green-700);
--color-pass: var(--color-green);
```

Reply:
Corrected this now